charts/memgraph/values.yaml

```diff
@@ -1,4 +1,8 @@
-image: memgraph/memgraph:latest
+image:
+  repository: memgraph/memgraph
+  # Overrides the image tag whose default is v{{ .Chart.AppVersion }}
+  tag: ""
```

Comment:
I think the default should match the appVersion, so 2.10.0 in this case

Reply:
That is what is already taking place

```
      containers:
        - name: memgraph
          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

We default to the app version if they do not explicitly set a image tag.